src/hooks/useDeviceMode.ts

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2022 - 2023 Gemeente Amsterdam
+import { useEffect, useState } from 'react'
+
+import { sizes } from '@amsterdam/asc-ui/lib/theme/default/breakpoints'
+
+export enum DeviceMode {
```

Comment:
Pin is using this enum as well. should this enum be extracted to a higher level?

Reply:
good one. I put is in `types/device' on toplevel